### ISSUE TITLE
How to get all potential CTDGs?

### ISSUE THREAD

**Issue by TelfordLab** [biofilos/ctdg_finder#5]
Hi!

This is a follow up on the previous discussion with @evolgenomology, but using the @TelfordLab handle for our own reference.
After successfully running the run_hmmer preprocessing steps using the Pfam-A database as target I am still wondering how to get the main python or julia script of ctdg_finder to work on the output, in particular without providing a specific gene family as target. What is the appropriate command for this?
Basically, what I was hoping to get is a report of all detected potential CTDGs and their genomic location, ideally with the corresponding Pfam IDs (or PANTHER if that is used) reported as well. Is this possible?

Thanks

Philipp (for the Telford Lab)

**Comment by biofilos** [comment]
you can use -p FAM to search for CTDGs of a specific family. FAM should be an accession ID from the HMMs used in the preprocessing steps. You can use that to identify CTDGs from all the families/domains you included by simply parsing those IDs and feeding them to CTDGFinder
